package.json

```diff
@@ -45,11 +45,11 @@
     "@docusaurus/module-type-aliases": "3.4.0",
     "@docusaurus/tsconfig": "3.4.0",
     "@types/react": "18.3.3",
-    "cspell": "8.11.0",
-    "husky": "9.1.1",
-    "lint-staged": "15.2.7",
+    "cspell": "8.13.1",
+    "husky": "9.1.4",
+    "lint-staged": "15.2.8",
     "prettier": "3.3.3",
-    "typescript": "5.5.3"
+    "typescript": "5.5.4"
```

Comment:
♻️ These are usually good to separate from content changes unless connected.

Reply:
Interesting, I'll check on that given that it wasn't my intention to change it. Sorry I missed it.